docs/source/conf.py

```diff
@@ -12,36 +12,39 @@
 #
 import os
 import sys
-sys.path.insert(0, os.path.abspath('.'))
-sys.path.insert(0, os.path.abspath('../../era5cli'))
 
-source_suffix = '.rst'
-master_doc = 'index'
+sys.path.insert(0, os.path.abspath("."))
+sys.path.insert(0, os.path.abspath("../../era5cli"))
+
+source_suffix = ".rst"
+master_doc = "index"
 
 # -- Project information -----------------------------------------------------
 
-project = 'era5cli'
-copyright = '2019, Netherlands eScience Center'
-author = 'era5cli team'
+project = "era5cli"
+copyright = "2019, Netherlands eScience Center"
+author = "era5cli team"
 
 # The full version, including alpha/beta/rc tags
-release = '1.3.1'
+release = "1.3.2"
```

Comment:
I realize I missed this with the recent new release! Perhaps it's good to update the contribution guide as well; adding conf.py to the documents that need to be updated when  bumping the version (and perhaps remove outdated information from that list, as some docs may not exist anymore as of #134).

Reply:
Good point, I'll add it to the contributing document.